cli/server_check_command.go

```diff
@@ -202,24 +210,26 @@ When set these settings will be used, but can be overridden using --waiting-crit
 	req.Flag("response-warn", "Warning threshold for response time").DurationVar(&c.msgWarn)
 
 	js := check.Command("jetstream", "Check JetStream account state").Alias("js").Action(c.checkJS)
-	js.Flag("mem-warn", "Warning threshold for memory storage, in percent").Default("75").IntVar(&c.jsMemWarn)
-	js.Flag("mem-critical", "Critical threshold for memory storage, in percent").Default("90").IntVar(&c.jsMemCritical)
-	js.Flag("store-warn", "Warning threshold for disk storage, in percent").Default("75").IntVar(&c.jsStoreWarn)
-	js.Flag("store-critical", "Critical threshold for memory storage, in percent").Default("90").IntVar(&c.jsStoreCritical)
-	js.Flag("streams-warn", "Warning threshold for number of streams used, in percent").Default("-1").IntVar(&c.jsStreamsWarn)
-	js.Flag("streams-critical", "Critical threshold for number of streams used, in percent").Default("-1").IntVar(&c.jsStreamsCritical)
-	js.Flag("consumers-warn", "Warning threshold for number of consumers used, in percent").Default("-1").IntVar(&c.jsConsumersWarn)
-	js.Flag("consumers-critical", "Critical threshold for number of consumers used, in percent").Default("-1").IntVar(&c.jsConsumersCritical)
+	js.HelpLong(multipleChecks + warnAndCritical + inversion)
+	js.Flag("mem-warn", "Warning threshold for memory storage, in percent of limit").Default("75").IntVar(&c.jsMemWarn)
+	js.Flag("mem-critical", "Critical threshold for memory storage, in percent of limit").Default("90").IntVar(&c.jsMemCritical)
+	js.Flag("store-warn", "Warning threshold for disk storage, in percent of limit").Default("75").IntVar(&c.jsStoreWarn)
+	js.Flag("store-critical", "Critical threshold for memory storage, in percent of limit").Default("90").IntVar(&c.jsStoreCritical)
```

Comment:
I believe this should be "disk storage", just like the store-warn flag. I know it was wrong before but could be fix it in this commit?